concepticondata/conceptlists/Seifart-2015-414.tsv

```diff
@@ -0,0 +1,415 @@
+Seifart-2015-414-15	15	shoulder	1482	SHOULDER	18
+Seifart-2015-414-16	16	arm	1673	ARM	19
+Seifart-2015-414-17	17	elbow	981	ELBOW	21
+Seifart-2015-414-18	18	hand	1277	HAND	22
+Seifart-2015-414-19	19	finger	1303	FINGER	23
+Seifart-2015-414-20	20	fingernail	1258	FINGERNAIL	24
+Seifart-2015-414-21	21	knee	1371	KNEE	27
+Seifart-2015-414-22	22	foot	1301	FOOT	29
+Seifart-2015-414-23	23	toe	1389	TOE	30
+Seifart-2015-414-24	24	skin	763	SKIN	31
+Seifart-2015-414-25	25	bone	1394	BONE	32
+Seifart-2015-414-26	26	blood	946	BLOOD	33
+Seifart-2015-414-27	27	heart	1223	HEART	34
+Seifart-2015-414-28	28	lungs	688	LUNG	35
+Seifart-2015-414-29	29	man	1554	MAN	38, 39
+Seifart-2015-414-30	30	woman	962	WOMAN	40
+Seifart-2015-414-31	31	people	789	PEOPLE	41
+Seifart-2015-414-32	32	husband	1200	HUSBAND	42
+Seifart-2015-414-33	33	wife	1199	WIFE	43
+Seifart-2015-414-34	34	father	1217	FATHER	44
+Seifart-2015-414-35	35	mother	1216	MOTHER	45
+Seifart-2015-414-36	36	baby,infant	1289	BABY	46
+Seifart-2015-414-37	37	old man	406	OLD MAN	47
+Seifart-2015-414-38	38	water	948	WATER	48
+Seifart-2015-414-39	39	river	666	RIVER	49
+Seifart-2015-414-40	40	swamp	1145	SWAMP	52
+Seifart-2015-414-41	41	fire	221	FIRE	56
+Seifart-2015-414-42	42	ash	646	ASH	57
+Seifart-2015-414-43	43	charcoal	1188	CHARCOAL	58
+Seifart-2015-414-44	44	firewood	10	FIREWOOD	60
+Seifart-2015-414-45	45	sky	1732	SKY	61
+Seifart-2015-414-46	46	rain	658	RAIN (PRECIPITATION)	62
+Seifart-2015-414-47	47	sun	1343	SUN	64
+Seifart-2015-414-48	48	moon	1313	MOON	65
+Seifart-2015-414-49	49	star	1430	STAR	66
+Seifart-2015-414-50	50	day	1225	DAY (NOT NIGHT)	67
+Seifart-2015-414-51	51	night	1233	NIGHT	68
+Seifart-2015-414-52	52	thunder	1150	THUNDER	69
+Seifart-2015-414-53	53	rainbow	1733	RAINBOW	71
+Seifart-2015-414-54	54	earth	1228	EARTH (SOIL)	72
+Seifart-2015-414-55	55	sand	671	SAND	74
+Seifart-2015-414-56	56	house	1252	HOUSE	75
+Seifart-2015-414-57	57	doorway	1567	DOOR	77
+Seifart-2015-414-58	58	seat,stool	2058	STOOL	78
+Seifart-2015-414-59	59	mat	195	MAT	79
+Seifart-2015-414-60	60	hammock	2019	HAMMOCK	80
+Seifart-2015-414-61	61	bed	1663	BED	81
+Seifart-2015-414-62	62	cooking pot	3653	COOKING POT	82
+Seifart-2015-414-63	63	cultivated clearing	212	FIELD	83
+Seifart-2015-414-64	64	village	930	VILLAGE	84
+Seifart-2015-414-65	65	path,trail	2457	PATH OR ROAD	85
+Seifart-2015-414-66	66	fish net	388	FISHNET	86
+Seifart-2015-414-67	67	fish hook	1074	FISHHOOK	87
+Seifart-2015-414-68	68	steel axe			88
+Seifart-2015-414-69	69	knife	1352	KNIFE	89
+Seifart-2015-414-70	70	canoe	1970	CANOE	90
+Seifart-2015-414-71	71	paddle	234	PADDLE	91
+Seifart-2015-414-72	72	club	1763	CLUB	92
+Seifart-2015-414-73	73	spear	945	SPEAR	93
+Seifart-2015-414-74	74	bow	994	BOW	94
+Seifart-2015-414-75	75	arrow	977	ARROW	95
+Seifart-2015-414-76	76	blowgun	432	BLOWGUN	96
+Seifart-2015-414-77	77	puma	1891	PUMA	99
+Seifart-2015-414-78	78	armadillo	1885	ARMADILLO	100
+Seifart-2015-414-79	79	dog	2009	DOG	101
+Seifart-2015-414-80	80	red deer			102
```

Comment:
We could think about adding RED DEER as a sub-concept of DEER. Please open an issue about this

See https://concepticon.clld.org/parameters/1936

Reply:
This is now pending for approval on the PR #1339.

Reply:
Thank you for mapping this, approved.